tests/Fixtures/Integration/misc/PHP7_4.test

```diff
@@ -9,7 +9,7 @@ PHP 7.4 test.
     "no_null_property_initialization": true,
     "no_unset_on_property": true,
     "pow_to_exponentiation": true,
-    "trailing_comma_in_multiline_array": {"after_heredoc": true},
+    "trailing_comma_in_multiline": {"after_heredoc": true},
```

Comment:
@keradus should this be also applied to version 2 line?

Reply:
would be nice, we don't need to use deprecations if not needed

Reply:
done: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5672